README.md

```diff
@@ -1,78 +1,157 @@
-
-### InterVar Classification Workflow
-This workflow is a critical component in generating scoring metrics needed to classify pathogenicity of variants.
-The workflow has three steps:
-1. Convert VCF input into ANNOVAR txt format
-1. Run ANNOVAR to gather needed scoring metrics
-1. Run InterVar
-
-### AutoPVS1
-An additional pathogenicty scoring tool.
-Documentation for this can be found [here](autopvs1/README.md)
\ No newline at end of file
+# AutoGVP: Automated Germline Variant Pathogenicity
+Jung Kim, Ammar S. Naqvi, Rebecca Kaufman, Miguel A. Brown, Ryan J. Corbett, Zhuangzhuang Geng, Zalman Vaksman, Phillip B. Storm, Adam C. Resnick, Jo Lynne Rokita, Douglas R. Stewart, Sharon J. Diskin
+
+## AutoGVP Workflow  
+![workflow](figures/germline-pathogenecity_flow.png)
+
+## Docker set-up
+
+### docker pull and run
+```bash
+docker pull pgc-images.sbgenomics.com/naqvia/germline-pathogenicity-assessment:latest
+docker run --platform linux/amd64 --name pathogenecity_anno -d -v $PWD:/home/rstudio/pathogenecity-assessment pgc-images.sbgenomics.com/naqvia/germline-pathogenicity-assessment:latest
+
+```
+### docker execute
+```bash
+docker exec -ti pathogenecity_anno bash
+```
+
+### cd to AutoGVP directory
+```bash
+cd /home/rstudio/AutoGVP
+```
+
+## How to Run
+
+### CAVATICA input ###
+1. Run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-germline-workflow/blob/v0.4.4/docs/GERMLINE_SNV_ANNOT_README.md) first.
+This workflow currently annotates variants with ClinVar (2022-05-07).
+2. Run the [Pathogenicity Preprocessing Workflow](https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing), which performs ANNOVAR with InterVar and AutoPVS1 annotations.
+3. Run AutoGVP.
+
+AutoGVP Requirements (recommended to place all in the `input/` folder):
+- VEP-, ANNOVAR-, and ClinVar annotated VCF file (`*VEP.vcf`)
+- ANNOVAR multianno file (`*hg38_multianno.txt`)
+- InterVar file (`*intervar.hg38_multianno.txt.intervar`)
+- AutoPVS1 file (`*autopvs1.txt`)
+- `variant_summary.txt`
+- `submission_summary.txt.gz` (can retrieve with `download_db_files.sh`)
+Note: the variant_summary and submission_summary files need to be uncompressed.
+
+Example run:
+
+```bash
+bash download_db_files.sh
+```
+```r
+Rscript 01-annotate_variants_custom_input.R --vcf <*.vcf> --multianno <*multianno.txt> --intervar <*hg38_multianno.txt.intervar> --autopvs1 <*autopvs1.txt --output <output_prefex> --submission input/variant_summary.txt --submission_summary input/submission_summary.txt
+```
+
+### Custom (non-CAVATICA) input ###
+1. Annotate the germline VCF with VEP.
+2. Run ANNOVAR with InterVar with the following options (to create the VCF input for AutoGVP).
+```perl
+perl table_annovar.pl input/test_hg38_selected_VEP_annotated.vcf hg38 --buildver hg38 --out test_hg38_selected --remove --protocol gnomad211_exome,gnomad211_genome --operation f,f --vcfinput
+```
```

Comment:
As above is for ANNOVAR annotation for gnomAD frequency and generating chrom,pos,ref,alt id to be merged with InterVar, we need another step where annotating file with InterVar

Reply:
Can you provide the command?

Reply:
python InterVar.py -b hg38 -i input/test_hg38_selected_VEP_annotated.vcf --input_type=VCF -o test_hg38_selected

Reply:
updated